clientv3/client.go

```diff
@@ -180,6 +189,15 @@ func newClient(cfg *Config) (*Client, error) {
 	client.Watcher = NewWatcher(client)
 	client.Auth = NewAuth(client)
 	client.Maintenance = &maintenance{c: client}
+	if cfg.Logger == nil {
+		// same as default std logger
+		client.logger = log.New(ioutil.Discard, "", 0)
```

Comment:
the comment is not accurate. probably just remove the comment. we should always comment why not what. the code itself is the what part.